src/components/shared/buttons/OutlinedToggleButton.tsx

```diff
@@ -23,10 +23,14 @@ const OutlinedToggleButton = styled(ToggleButton)(({
         [`&.${toggleButtonClasses.selected}`]: {
             'backgroundColor': theme.palette[colorKey].alpha_12,
             'border': `1px solid ${theme.palette[colorKey].alpha_50}`,
-            'color': theme.palette[colorKey].dark,
+            'color':
+                theme.palette.mode === 'light'
+                    ? theme.palette[colorKey].dark
+                    : theme.palette[colorKey].main,
             '&:hover': {
                 border: `1px solid ${
```

Comment:
We eventually might need to look at not using the shorthand settings when doing more complex settings and just break the properties up.